graphql/mappers/beta-banner-opt-out.js

```diff
@@ -2,9 +2,12 @@ import clientQuery from '../client'
 
 export async function getBetaBannerContent() {
   const queryOptOut = require('../queries/beta-banner-opt-out.graphql')
+  const queryDictionary = require('../queries/dictionary.graphql')
   const resOptOut = await clientQuery(queryOptOut)
+  const resDictionary = await clientQuery(queryDictionary)
 
   const resOptOutContent = resOptOut.data.schContentv1ByPath.item || {}
+  const resDictionaryContent = resDictionary.data.dictionaryV1List.items || {}
```

Comment:
There is a way to query once and pass around a reference.
I'm not positive but I believe custom hooks handle this issue for generic content

Reply:
As per our conversation, something to revisit. Currently we're only using the dictionary for the one term, but the opportunity is there to use it more.